app/src/main/java/de/westnordost/streetcomplete/quests/smoking/SmokingAllowed.kt

```diff
@@ -0,0 +1,8 @@
+package de.westnordost.streetcomplete.quests.smoking
+
+enum class SmokingAllowed(val osmValue: String) {
+    YES("yes"),
+    OUTDOOR("outdoor"),
```

Comment:
The OSM tagging is `outside` in this case: https://wiki.openstreetmap.org/wiki/Key:smoking

My comment about renaming outside to outdoor was only targeted at the `isAlreadyOutside` variable, as it refers to `outdoor_seating` there. Sorry for not being clear!

Reply:
should be fixed by https://github.com/streetcomplete/StreetComplete/pull/3784/commits/1633c940b7f01570beb4c54addd52ec52df0a3fb